tests/integration/test_storage_mysql/test.py

```diff
@@ -418,6 +418,9 @@ def test_predefined_connection_configuration(started_cluster):
     ''')
     assert (node1.query(f"SELECT count() FROM test_table").rstrip() == '100')
 
+    assert 'Connection pool cannot have zero size' in node1.query_and_get_error(f"SELECT count() FROM mysql(mysql1, table='test_table', connection_pool_size=0)")
+    assert 'Connection pool cannot have zero size' in node1.query_and_get_error(f"SELECT count() FROM mysql(mysql4)")
```

Comment:
Maybe also check that if we add `connection_pool_size=1` it overrides setting from config and works.